src/jenkins_job_insight/main.py

```
@@ -2529,19 +2647,49 @@ async def _execute_rp_push(
-            *(
-                get_history_classification(job_id, name, scope_name, scope_build)
-                for name in unique_test_names
+        try:
+            classifications = await asyncio.gather(
+                *(
+                    get_history_classification(job_id, name, scope_name, scope_build)
+                    for name in unique_test_names
+                )
             )
-        )
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)